manager/ui/war/src/main/java/io/apiman/manager/ui/server/impl/UIConfig.java

```diff
@@ -13,9 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.apiman.manager.ui.server;
+package io.apiman.manager.ui.server.impl;
```

Comment:
To avoid that the classloader of apiman-common-util imports the impl class located here previsouly "io.apiman.manager.ui.server" as we will use another class for the osgi bundle of the apiman manager ui